packages/web/src/components/upload-confirmation-modal/UploadConfirmationModal.tsx

```diff
@@ -47,11 +48,13 @@ export const UploadConfirmationModal = () => {
         <ModalTitle icon={<IconUpload />} title={messages.title} />
       </ModalHeader>
       <ModalContent>
-        <ModalContentText>
-          {hasPublicTracks
-            ? messages.publicDescription
-            : messages.hiddenDescription}
-        </ModalContentText>
+        <Flex justifyContent='center'>
+          <Text variant='body' size='l' textAlign='center'>
```

Comment:
imo better to make this change in `harmony/ModalContentText`, but would also need to validate all modals

Reply:
we could also leave it left aligned for now until we take on the modal updates

Reply:
talked to design and we're going to just do this as a 1-off for now, but do the sweeping change later